frogs-data.js

```diff
@@ -0,0 +1,17 @@
+export const frogs = [
+    {
+        name: 'Bullfrog',
+        img: 'bullfrog.jpg',
+        habitat: ['ponds', 'swamps', 'marshes'],
```

Comment:
good job attempting more complicated shape of object

edit: sounds kind of passive aggressive "oh nice attempt" just meaning good on you for doing something more complicated